TODO.md

```diff
@@ -0,0 +1,3 @@
+- labwc config wird nicht korrekt aktualisiert on system activation
+  - Korrekter, wenn das image nicht weggeschmissen wird, kriegt man nicht alle auswirkungen einer neuen version des flakes
```

Comment:
I've noticed this too, what's your guess on why that is?

Reply:
Not really. I suspect tit might be a compound error of sorts. As in: error in the script, not activation script breaks without noise, and thus it looks like system activation is not run every time? But I really haven't spent time investigating this.